glib-macros/src/lib.rs

```diff
@@ -525,6 +525,35 @@ pub fn closure_local(item: TokenStream) -> TokenStream {
 /// }
 /// ```
 ///
+/// When using [`Properties`] with an enum that derives [`Enum`], the default
```

Comment:
This is not specific to enums that are derived via `glib::Enum` but also other existing enums, e.g. provided by C libraries.

I think it would make more sense to move this to the documentation of the `Properties` macro or what do you think?

Reply:
Thanks :)

And maybe leave a small paragraph here that points to the Properties docs? "When used with the properties macro, you need to blablablabla link to the docs of that"